RWFramework/RWFramework/Playlist/Playlist.swift

```diff
@@ -405,6 +405,16 @@ extension Playlist {
             $0.playNext(premature: true)
         }
     }
+    public func getCurrentAssets() -> [Asset] {
```

Comment:
Again, we don't have a backing field here, but I'd use a dynamic property instead of a method here. Something like:
```
public var currentAssets: [Asset] {
  tracks?.map { $0.currentAsset } ?? []
}
```

Reply:
Secondly, less important, how about the name `currentlyPlaying` for clarity?